src/app/home/HomeComponent.tsx

```diff
@@ -41,18 +64,43 @@ const HomeComponent: React.FunctionComponent<IHomeComponentProps> = ({
       </NavList>
     </Nav>
   );
+  const isWelcomeScreen = history.location.pathname === '/welcome';
+  const [isShowNav, setIsShowNav] = useState(false);
+
+  useEffect(() => {
```

Comment:
I think we don't actually need to use `isManagedSidebar` and a `useEffect` here at all, we can just not render the `<PageSidebar>` at all if `isWelcomeScreen` is true and let the sidebar manage itself when it is rendered.

Either that, or we can just have `isNavOpen` be `!isWelcomeScreen && isShowNav` and we still wouldn't need the useEffect.

I'll implement this too if you want.

Reply:
Resolved in 3ddae45d545758d1bb7d0085b511473de6176d9f